tests/unit/v1/test_collection.py

```diff
@@ -32,10 +32,18 @@ def _make_one(self, *args, **kwargs):
 
     @staticmethod
     def _get_public_methods(klass):
-        return set(
-            name
-            for name, value in six.iteritems(klass.__dict__)
-            if (not name.startswith("_") and isinstance(value, types.FunctionType))
+        return set().union(
```

Comment:
why this change?

Reply:
The previous test was testing the functions that exist in `CollectionReference`. Now that we've pulled some of those out to `BaseCollectionReference`, we need to test the public interface, which means including the functions that exist in the parent. This was my way of solving that problem.